src/modules/Touchable.js

```diff
@@ -0,0 +1,267 @@
+const React = require('react');
+const TimerMixin = require('react-timer-mixin');
+
+const ensurePositiveDelayProps = (props) => {
+  // invariant(
+  //   !(props.delayPressIn < 0 || props.delayPressOut < 0 || props.delayLongPress < 0),
+  //   'Touchable components cannot have negative delay properties'
+  // );
+};
+
+const { PropTypes } = React;
+
+const InsetPropType = PropTypes.shape({
+  top: PropTypes.number,
+  left: PropTypes.number,
+  bottom: PropTypes.number,
+  right: PropTypes.number,
+});
+
+
+// NOTE(lmr): this is a huge hack right now, and prevents anything from being clickable more than
+// twice per second, but the alternative is so bad right now. Need to figure out how to fix the
+// responder plugin later and fix this.
+const THROTTLE_MS = 500;
+
+function throttle(fn, throttleMs) {
+  let lastCall = null;
+
+  return function (...args) {
+    const now = new Date;
+    if (lastCall === null || (now - lastCall > throttleMs)) {
+      fn.apply(this, args);
+      lastCall = new Date;
+    }
+  };
+}
+
+/**
+ * A wrapper for making views respond properly to touches.
+ * On press down, the opacity of the wrapped view is decreased, dimming it.
+ * This is done without actually changing the view hierarchy, and in general is
+ * easy to add to an app without weird side-effects.
+ *
+ * Example:
+ *
+ * ```
+ * renderButton: function() {
+ *   return (
+ *     <Touchable press={anim} onPress={this._onPressButton}>
+ *       <Image
+ *         style={styles.button}
+ *         source={require('image!myButton')}
+ *       />
+ *     </Touchable>
+ *   );
+ * },
+ * ```
+ */
+const Touchable = (
+  Animated,
```

Comment:
i'm going to try and get this merged as is, but we should have a broader discussion about animated, especially as it pertains to size, as we want to try and make RP as small as possible especially on web